book/src/testnet-validator.md

```diff
@@ -9,25 +9,26 @@ DEPOSITS WITH LIGHTHOUSE. 🚨🚨🚨**
 
 ## Supported Testnets
 
-Lighthouse supports four testnets:
+Lighthouse supports the "mainnet" network and four test networks:
 
 - [Medalla](https://github.com/goerli/medalla/tree/master/medalla) (default)
 - [Pyrmont](https://github.com/protolambda/pyrmont)
 - [Spadina](https://github.com/goerli/medalla/tree/master/spadina) (deprecated)
 - [Altona](https://github.com/goerli/medalla/tree/master/altona) (deprecated)
 
-When using Lighthouse, the `--testnet` flag selects a testnet. E.g.,
+When using Lighthouse, the `--network` flag selects a network. E.g.,
 
 - `lighthouse` (no flag): Medalla.
```

Comment:
```suggestion
- `lighthouse` (no flag): Mainnet.
```

Reply:
Good point, I'm going to skip this here since I have another PR coming that fixes it.